readthedocs/projects/management/commands/dump_project_remote_repo_relation.py

```diff
@@ -0,0 +1,45 @@
+import json
+
+from django.core.management.base import BaseCommand
+
+from readthedocs.projects.models import Project
+
+
+class Command(BaseCommand):
+    help = "Dump Project and RemoteRepository Relationship in JSON format"
+
+    def handle(self, *args, **options):
+        data = []
+
+        queryset = Project.objects.filter(
+            remote_repository__isnull=False,
+        ).values_list('id', 'remote_repository__json').distinct()
+
+        for project_id, remote_repository__json in queryset:
+            try:
+                json_data = json.loads(remote_repository__json)
+                # GitHub and GitLab uses `id` and Bitbucket uses `uuid`
+                # for the repository id
+                remote_id = json_data.get('id') or json_data.get('uuid')
+
+                if remote_id:
+                    data.append({
+                        'remote_id': remote_id,
+                        'project_id': project_id
+                    })
```

Comment:
Can we save the `RemoteRepository.users.first().username` here as well? This is not _required_ but could help us to re-sync users with linked projects quickly.

Reply:
Also, let's save the `full_name` and `provider` as well. That will help us to debug things in case something breaks.